ui-tests/src/test/java/org/openmrs/reference/RecordMergeIssueTest.java

```diff
@@ -31,7 +33,7 @@
     @Before
     public void setUp() throws Exception {
         homePage = new HomePage(page);
-        assertPage(homePage);
```

Comment:
The function assertPage(homePage) is vital. 

Add `waitForPage() `function on` homePage` object to ensure the page is loaded for assertion to be effective ie `assertPage(homePage.waitForPage());` otherwise the function asserts an empty page resulting into `java.lang.AssertionError.`